File: sdk/approute-sdk/python/approute/models/steam_currency.py
from datetime import datetime

from approute.models._base import _Base


class SteamCurrencyRate(_Base):
    quote_currency_code: str
    rate: str
    provider_created_at: datetime | None = None
    fetched_at: datetime | None = None


class SteamCurrencyRatesResponse(_Base):
    base_currency_code: str
    items: list[SteamCurrencyRate]

File: sdk/approute-sdk/javascript/src/models/steam-currency.ts
/**
 * A single Steam currency exchange rate.
 */
export interface SteamCurrencyRate {
  quoteCurrencyCode: string;
  rate: string;
  providerCreatedAt?: string;
  fetchedAt?: string;
}

/**
 * Response containing Steam currency exchange rates.
 */
export interface SteamCurrencyRatesResponse {
  baseCurrencyCode: string;
  items: SteamCurrencyRate[];
}
